From 0e1d8d0e06108f2887c3147c93049de63b475f8a Mon Sep 17 00:00:00 2001
From: eyjhb <eyjhbb@gmail.com>
Date: Sat, 25 Oct 2025 12:02:14 +0200
Subject: [PATCH] get_ek_field_mapping: removed check on dynamic

This field is no longer available in wireshark version 4.6.0, and it
seems like it does not make sense to check for it anyways.
---
 src/pyshark/tshark/tshark.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: src/pyshark/tshark/tshark.py
===================================================================
--- src.orig/pyshark/tshark/tshark.py
+++ src/pyshark/tshark/tshark.py
@@ -139,7 +139,7 @@ def get_ek_field_mapping(tshark_path=Non
         mapping,
         object_pairs_hook=_duplicate_object_hook)["mappings"]
     # If using wireshark 4, the key "mapping" contains what we want,
-    if "dynamic" in mapping and "properties" in mapping:
+    if "properties" in mapping:
         pass
     # if using wireshark 3.5 to < 4 the data in "mapping.doc",
     elif "doc" in mapping:
